plugins/module_utils/aci.py

```diff
@@ -1592,7 +1597,14 @@ def exit_json(self, filter_existing=None, **kwargs):
         if "state" in self.params:
             self.original = self.existing
             if self.params.get("state") in ("absent", "present"):
-                self.get_existing()
+                if self.no_verification:
+                    if self.result["changed"]:
+                        self.result["current_verified"] = False
+                        self.existing = []
```

Comment:
Should we provide alternative output into current? Perhaps the send or proposed values for new config, and a merged view of first retrieve with updates send?

Reply:
I thought about and experimented with merging previous (if any) with sent to at least show some current. However, I think that might get a little inconsistent and won't show default values so I decided it was best not to include any current. I believe if the user is configuring no_verification they would be aware of the limitation and not really intending to use current anyway.

Reply:
1. I believe it would be useful to show "current" with some info and also document that default values are omitted with no_verification because the info in "current" may be used by users for purposes unknown to us? Once the users get used to the faster speed in deployments they would want the best of both the worlds.

2. If "changed" then we pass "proposed" to "current" else "previous" to "current".

Reply:
I think the general consensus now is to put something in current even if defaults aren't shown. I'll make the change and modify the warning.